metaflow/plugins/aws/batch/batch.py

```diff
@@ -345,9 +345,23 @@ def _print_available(tail, stream, should_persist=False):
         # TODO: AWS CloudWatch fetch logs
 
         if self.job.is_crashed:
-            msg = next(msg for msg in 
-                [self.job.reason, self.job.status_reason, 'Task crashed.']
-                 if msg is not None)
+            if self.job.status_code == 1:
+                msg = next(msg for msg in ['Task failed with a general error. You may have a miscellaneous error(s), '
```

Comment:
I am not sure if I follow the lambda here. The list is a singleton here and below. Also, have you looked into AWS Batch error codes? I don't think the status codes map to the descriptions that you have written.

Reply:
I couldn't find any documentation on AWS Batch error codes. You gave me a link to exit codes earlier and that was what I used to write the descriptions. Let me know if I should use another source.

Also, it would be great if you could give me a bit more detail on the error with my If statement. Thanks!